docs/src/adoc/index.adoc

```diff
@@ -3409,7 +3409,8 @@ jdbi.installPlugin(KotlinPlugin());
 ----
 
 Result set mapper also supports `@ColumnName` annotation that allows to specify
```

Comment:
Aside: some of these "ResultSet" / "result set" mentions seem to be referring to `ResultSetMapper`, which is the v2 name for `RowMapper`.

Reply:
I'll update those to "RowMapper".